api/src/main/java/org/openmrs/Diagnosis.java

```diff
@@ -17,6 +17,23 @@
  */
 public class Diagnosis extends BaseChangeableOpenmrsData {
 
+	/**
+	 * The diagnosis order shows if a diagnosis is primary or secondary diagnosis
+	 *
+	 * @since 2.2
+	 */
+	public enum DiagnosisOrder {
```

Comment:
This should not be included

The old implementation in the emrapi module has an enum for this, but now that we're implementing it openmrs-core we want to be more flexible, so the "Integer rank" field replaces diagnosisOrder.

See the parent JIRA issue.

Reply:
Thanks @djazayeri. Also, how can I determine if a diagnosis is primary or secondary?